src/editor-page.jsx

```diff
@@ -17,7 +17,9 @@ var ViewportResizer = require("./components/viewport-resizer.jsx");
 
 var EditorPage = React.createClass({
     propTypes: {
+        /* eslint-disable react/forbid-prop-types */
         answerArea: React.PropTypes.object,
```

Comment:
The ones I marked as shape({}) are unknowable, where as these are knowable but just unknown right now (</Rumsfeld>) and I think it's valuable to document the distinction, but it's probably better if I just do that with TODO comments.